ui/src/components/Share/index.vue

```diff
@@ -125,7 +125,7 @@ const shareLinkRequest = reactive({
 });
 
 const shareURL = computed(() => {
-  return shareId.value ? `${BASE_URL}/koko/share/${shareId.value}/` : t('NoLink');
+  return shareId.value ? `${BASE_URL}/luna/share/${shareId.value}/` : t('NoLink');
 });
 
 const mappedUserOptions = computed(() => {
```

Comment:
There seems to be no issues related to this code snippet. It appears correctly written and functioning.

However, please note some important points:

1. The `t('NoLink')` calls should likely include translations for specific countries/regions that use different language packs if applicable. For example, "NoShare" could potentially need translation for Chinese-speaking platforms (e.g., Alibaba Cloud).

This is assuming we're working with an open-source repository where the current version can't specify such localized strings in a straightforward way due to limitations like API usage or data storage policies of services used by it.